organizations.py

```diff
@@ -0,0 +1,150 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session
+
+from __init__ import app
+from connection_initializer import init_tcp_connection_engine
+from connection_initializer import init_unix_connection_engine
+from connection_initializer import load_secrets
+
+Base = declarative_base()
+
+
+class Organization(Base):
+    """Model of Organizations for SQLAlchemy."""
+
+    __tablename__ = "organizations"
+    __table_args__ = ({"mysql_charset": "utf8mb4"})
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    name = Column(String, nullable=False)
+    login_id = Column(String, nullable=False)
+    login_password = Column(String, nullable=False)
+    airtable_base_id = Column(String, nullable=False)
+    airtable_api_key = Column(String, nullable=False)
+    amazon_access_key = Column(String, nullable=False)
+    amazon_secret_key = Column(String, nullable=False)
+    amazon_partner_tag = Column(String, nullable=False)
+    database_name = "master_data"
+
+
+class OrganizationManager:
+    """OrganizationManager for Connect Cloud SQL."""
+
+    database_engine = None
+    database_config = {
+        "echo": True,
+        "pool_size": 5,
+        "max_overflow": 2,
+        "pool_timeout": 30,
+        "pool_recycle": 1800
+    }
+    secrets = {"CLOUD_SQL_CONNECTION_NAME": "", "DB_USER": "", "DB_PASSWORD": "", "PASS_PHRASE": ""}
+
+    def __init__(self):
+        """Initialize OrganizationManager."""
+
+        OrganizationManager.secrets = load_secrets(OrganizationManager.secrets)
+        # print(OrganizationManager.secrets["PASS_PHRASE"])
+
+        if app.config["IS_LOCAL"]:
+            self.database_engine = init_tcp_connection_engine(
+                OrganizationManager.secrets, Organization.database_name, OrganizationManager.database_config)
+        else:
+            self.database_engine = init_unix_connection_engine(
+                OrganizationManager.secrets, Organization.database_name, OrganizationManager.database_config)
+
+        Base.metadata.create_all(bind=self.database_engine)
+
+    def authenticate_organizations(self, login_id: str, login_password: str) -> int:
+        """Verify login.
+
+        Args:
+            login_id (str): Uniquely identifiable user ID registered with organizations.
+            login_password (str): Passwords for registered users in organizations.
+
+        Returns:
+            Int<ID or 0>: The integer value of the ID in the organization.
+        """
+
+        pass_phrase = OrganizationManager.secrets["PASS_PHRASE"]
+        session = Session(bind=self.database_engine)
+        authentications_statement = text(
+            "select id from organizations "
+            "where login_id = '%s' and AES_DECRYPT(login_password, SHA2('%s', 512)) = '%s'"
+            % (login_id, pass_phrase, login_password)
+        )
+        result = session.execute(authentications_statement).fetchone()
+        if (result is None):
```

Comment:
```suggestion
        return result.id if result else 0
```
79行目以降はこんな感じでまとめられそう

Reply:
そうですね。
こちらの方がシンプルなのでこちらにします。
ありがとうございます。